### PR TITLE
EVSRESTAPI-524: Verify that getting all NCIt concepts from search api works

### PR DESCRIPTION
**Pull request by bcarlsenca**

### CODE REVIEW

File: src/test/java/gov/nih/nci/evs/api/controller/SearchControllerTests.java [hunk 1]
```diff
@@ -4015,6 +4021,115 @@ public void testSparqlPaging() throws Exception {
...
+      total = list.getTotal();
+      if (list.getConcepts() == null || list.getConcepts().isEmpty()) {
+        log.info("  done reading");
+        break;
+      }
+      codes.addAll(list.getConcepts().stream().map(c -> c.getCode()).collect(Collectors.toList()));
+      fromRecord += pageSize;
+    }
+
+    // Verify no duplicate codes
```
Comment (peter-va) [inline src/test/java/gov/nih/nci/evs/api/controller/SearchControllerTests.java:4126]: Is this supposed to test that the sort parameter worked properly? Right now it seems to be doing the exact same checks as the other added test with the only difference being the sort parameter, which isn't checked.

**Review by peter-va — CHANGES_REQUESTED** [review]
Needs to pass the spotlessJava check, and a quick clarification requested on one thing regarding the testSearctAllNcitWithSort (I think that's a typo) test. Otherwise looks good.

File: src/test/java/gov/nih/nci/evs/api/controller/SearchControllerTests.java (hunk 1 above)
Comment (bcarlsenca) [reply src/test/java/gov/nih/nci/evs/api/controller/SearchControllerTests.java:4126]: Good question. I'll put a comment to explain this. We don't care about checking sort order here, but a search without a sort parameter uses "rank" sort, so I'm checking that both "rank" sort and a chosen fielded sort both work properly (as I've seen situations with elasticsearch where they behave differently)

**Comment by peter-va** [comment]
Alright just needs to pass the spotlessJava check now and it'll be ready to merge @bcarlsenca

**Review by peter-va — APPROVED** [review]

**Pull request merged**